ocw/lib/EC2.py

```diff
@@ -117,6 +117,32 @@ def cleanup_snapshots(self):
                         else:
                             raise ex
 
+    def cleanup_volumes(self):
+        cleanup_ec2_max_volumes_age_days = PCWConfig.get_feature_property('cleanup', 'ec2-max-volumes-age-days',
+                                                                          self._namespace)
+        if cleanup_ec2_max_volumes_age_days < 0:
+            return
+        delete_older_than = date.today() - timedelta(days=cleanup_ec2_max_volumes_age_days)
+        for region in self.all_regions:
+            response = self.ec2_client(region).describe_volumes()
+            for volume in response['Volumes']:
+                if datetime.date(volume['CreateTime']) < delete_older_than:
+                    self.log_info("Deleting volume {} in region {} with CreateTime={}", volume['VolumeId'],
```

Comment:
I would move this line to 135, so we do not log "delete volume" but we don't do it.